Bottom: 68f6ef22b267a93ea21548bdaf6005313fcd3b83
Top:    16a66fb6f769e01f6448d4cbd9f1c882b90ee6c9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 11:42:39 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index cec5372..b263b9f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
